### PR TITLE
exo run port assignment

### PR DESCRIPTION
**Pull request by hugoesthere**
Resolves https://github.com/Originate/exosphere/issues/726

### CODE REVIEW

File: src/types/service_development_config.go [hunk 1]
```diff
@@ -4,4 +4,5 @@ package types
 type ServiceDevelopmentConfig struct {
 	Dependencies []DevelopmentDependencyConfig
 	Scripts      map[string]string `yaml:",omitempty"`
+	Ports        []string          `yaml:",omitempty"`
```
Comment (charlierudolph) [inline src/types/service_development_config.go]: We want to keep this as a single port for now. We will be getting to multiple ports soon enough.

Comment (hugoesthere) [reply src/types/service_development_config.go]: But I think we get this for free, no? All we're doing is creating another entry in the `ports` block of the yml file which docker-compose handles for us

Comment (charlierudolph) [reply src/types/service_development_config.go]: For multiple ports @alexdavid and I had talked about:
```yml
ports: 
  key1: 3000
  key2: 4000
```
where the keys will be used in the env variables we pass to other services. Thus wanted to address this after we have service env variables

Comment (hugoesthere) [reply src/types/service_development_config.go]: Ah, ok. How are we managing env vars for ports right now? Is it ok to just have `port: 80` in service.yml?

Comment (charlierudolph) [reply src/types/service_development_config.go]: We aren't doing the env vars for ports yet (that will be the next task). Yes, for now please update this so you would add `port: 80` (separate ports under dev / production)

**Review by charlierudolph — APPROVED** [review]
Merge on green

**Pull request merged**